backend/routers/lora_management_router.py

```
@@ -67,9 +68,81 @@ async def create_replicate_model(
+            response = await client.post(upload_request_url, headers=headers, json=payload)
+            response.raise_for_status()
+            
+            upload_data = response.json()
+            logger.info("Successfully received upload URLs from Replicate.")
+
+            upload_url = upload_data.get("upload_url")
+            serving_url = upload_data.get("serving_url")
+
+            if not upload_url or not serving_url:
+                missing_keys = []
+                if not upload_url: missing_keys.append("upload_url")
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)